velox/exec/PartitionedOutputBufferManager.cpp

```diff
@@ -275,12 +371,83 @@ BlockingReason PartitionedOutputBuffer::enqueue(
                  : BlockingReason::kNotBlocked;
 }
 
+void PartitionedOutputBuffer::enqueueBroadcastOutputLocked(
+    std::unique_ptr<SerializedPage> data,
+    std::vector<DataAvailable>& dataAvailableCbs) {
+  VELOX_DCHECK(isBroadcast());
+  VELOX_CHECK_NULL(arbitraryBuffer_);
+  VELOX_DCHECK(dataAvailableCbs.empty());
+
+  std::shared_ptr<SerializedPage> sharedData(data.release());
+  for (auto& buffer : buffers_) {
+    if (buffer != nullptr) {
+      buffer->enqueue(sharedData);
+      dataAvailableCbs.emplace_back(buffer->getAndClearNotify());
```

Comment:
Can we still call it "dataAvailableCallbacks", it's a little bit confusing with the new name

Reply:
Cbs is a common abbreviations for Callbacks in c++. I am ok either way and sometime I choose cbs if the variable name is already too long.